BinaryObjectScanner.Protection/Macrovision.cs

```diff
@@ -48,32 +52,79 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (sections == null)
                 return null;
 
+            // Check for specific indications for individual Macrovision protections.
+            List<string> resultsList = new List<string>();
+
             // Check for generic indications of Macrovision protections first.
             string name = pex.FileDescription;
 
+            // Present in "secdrv.sys" files found in SafeDisc 2.80.010+.
+            if (name?.Equals("Macrovision SECURITY Driver", StringComparison.OrdinalIgnoreCase) == true)
+                resultsList.Add($"Macrovision Security Driver {GetSecDrvExecutableVersion(pex)}");
+
             // Found in hidden resource of "32bit\Tax02\cdac14ba.dll" in IA item "TurboTax Deluxe Tax Year 2002 for Wndows (2.00R)(Intuit)(2002)(352282)".
-            // TODO: Fix File Description not getting properly pulled for this executable.
             // Known versions:
             // 4.16.050 Windows NT 2002/04/24
             if (name?.Equals("Macrovision RTS Service", StringComparison.OrdinalIgnoreCase) == true)
-                return $"Macrovision RTS Service {pex.FileVersion}";
-
-            // Check for specific indications for individual Macrovision protections.
+                resultsList.Add($"Macrovision RTS Service {pex.FileVersion}");
+            string match;
```

Comment:
Nit: Odd spacing for `match` here. Prefer to either put it down at the CDS check and use another name within the `if` and `else` blocks. An example would be using `sectionMatch` or something similar.